File: sentimental-mario-less/mario.py
# TODO
from cs50 import get_int

while True:
    try:
        height = get_int("Height: ")
        if height <= 8 and height >= 1:
            break
    except:
        ValueError("Enter an integer between 1-8 inclusive.")

for i in range(height):
    print((height - i - 1) * " ", end="")
    print((i + 1) * "#", end="")
    print()

File: mario-less/mario.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{
    int i, j, k, l;
    do
    {
        l = get_int("Height: ");
    }
    while (l <= 0 || l >= 9);

    for (i = 1; i <= l; i++)
    {
        for (k = l - 1; k >= i; k--)
        {
            printf(" ");
        }
        for (j = 1; j <= i; j++)
        {
            printf("#");
        }
        printf("\n");
    }
}